manim/animation/indication.py

```diff
@@ -142,50 +144,50 @@ def finish(self):
             submob.pointwise_become_partial(start, 0, 1)
 
 
+@attr.s(auto_attribs=True, eq=False)
 class ShowCreationThenDestruction(ShowPassingFlash):
-    CONFIG = {
-        "time_width": 2.0,
-        "run_time": 1,
-    }
+    time_width: float = 2.0
+    run_time: float = 1
 
 
+@attr.s(auto_attribs=True, eq=False)
 class ShowCreationThenFadeOut(Succession):
-    CONFIG = {
-        "remover": True,
-    }
+    remover: bool = True
 
-    def __init__(self, mobject, **kwargs):
-        super().__init__(ShowCreation(mobject), FadeOut(mobject), **kwargs)
+    def __attrs_post_init__(self):
+        self.animations = [ShowCreation(self.mobject), FadeOut(self.mobject)]
+        Succession.__attrs_post_init__(self)
 
 
+@attr.s(auto_attribs=True, eq=False)
 class AnimationOnSurroundingRectangle(AnimationGroup):
-    CONFIG = {
-        "surrounding_rectangle_config": {},
-        # Function which takes in a rectangle, and spits
-        # out some animation.  Could be some animation class,
-        # could be something more
-        "rect_animation": Animation,
-    }
+    surrounding_rectangle_config: tp.Dict = attr.ib(default=attr.Factory(dict))
+    # Function which takes in a rectangle, and spits
+    # out some animation.  Could be some animation class,
+    # could be something more
+    rect_animation: type = Animation
+    kwargs_animations: tp.Dict = attr.ib(default=attr.Factory(dict))
```

Comment:
note that we generally use tp.Dict to specify the types of the dict's keys and values: tp.Dict[K, V]
```suggestion
    kwargs_animations: dict = attr.ib(default=attr.Factory(dict))
```